pkgs/development/python-modules/py-slvs/default.nix

```
@@ -0,0 +1,37 @@
+{ lib, blender, python3Packages, pkgs, fetchurl }:
+
+python3Packages.buildPythonPackage rec {
+  pname = "py-slvs";
+  version = "1.0.6";
+
+  src = python3Packages.fetchPypi {
+    pname = "py_slvs";
+    version = "1.0.6";
```

If you're using `rec` then you can inherit the version value from the outer attrset, making the package less work to update (only 1 place to change).
```suggestion
    inherit version;
```